Add Aidan Khelil's Profile to IRIS-HEP website

akhelil opened the pull request.

akhelil committed:
Add files via upload

akhelil committed:
Create akhelil.md

pelmer commented:
Hi Aidan, will you fill in the "TO-EDIT" sections at the bottom of the .md file? And perhaps remove the Iris-hep-logo.png from the list.

akhelil committed:
Update akhelil.md

rct225 committed:
Merge branch 'master' into master

akhelil committed:
Update akhelil.md to add some bio info

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

akhelil commented:
> Hi Aidan, will you fill in the "TO-EDIT" sections at the bottom of the .md file? And perhaps remove the Iris-hep-logo.png from the list.

I added some info to the TO-EDIT sections; I removed a couple, as I felt they would make me slightly redundant.
